src/deno_json/mod.rs

```diff
@@ -568,6 +586,7 @@ pub struct ConfigFileJson {
   pub lock: Option<Value>,
   pub exclude: Option<Value>,
   pub node_modules_dir: Option<bool>,
```

Comment:
We should add a workspace diagnostic for when someone sets the `nodeModulesDir` option.